src/styles/components/form-elements/form-group/styles.less

```diff
@@ -10,6 +10,10 @@
       }
     }
 
+    .extra-bottom {
```

Comment:
I REALLY dislike we need to keep adding this kinda of css all over.

Reply:
We shouldn't need to touch CSS for things like this. It should all be magic if the markup is accurate.

Reply:
Thanks for commenting @leemunroe 

@ahoskins do you want to take another look on it and see if you can accomplish the same or close resul without this please?

Reply:
Yes I also really dislike this! I remember looking previously and not finding a way existing. Will look again.

Reply:
The issue is the vertical margins come from the heading. And in this case the next field in the form is just a <label> and class "form-group-heading", which does not provide any top margin.

<img width="811" alt="screen shot 2018-02-09 at 3 29 12 pm" src="https://user-images.githubusercontent.com/1527504/36054954-0b0ead0c-0dae-11e8-8946-0f1b2c78c300.png">

Reply:
Fixed this by using .pod from cnvs, and flush on all sides except the bottom. 👍